### PR TITLE
Pull/1301

### PR DESCRIPTION
**Pull request by bentito**
After running `make metering-manifests`

### CODE REVIEW

File: charts/metering-ansible-operator/values.yaml [hunk 1]
```diff
@@ -227,21 +227,25 @@ olm:
...
 
       ### Core capabilities
 
-      * **Chargeback/Showback** - Break down the reserved and utlized resources requested by applications.
+      * **Long term storage** - Break down the reserved and utlized resources requested by applications and store the information as long as needed.
 
       * **Pod, Namespace & Cluster Reports** - Built in reports exist to break down CPU and RAM in any way you desire.
-
-      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly
+
```
Comment (timflannagan1) [inline charts/metering-ansible-operator/values.yaml:239]: This is rendering as a random space.

File: charts/metering-ansible-operator/values.yaml [hunk 2]
```diff
@@ -227,21 +227,25 @@ olm:
     minKubeVersion: "1.18.0"
     displayName: Metering
     description: |
-      The Metering Operator is a chargeback and reporting tool to provide accountability for how resources are used across a cluster. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster wide. The Metering Operator is a part of the [Kubernetes Reporting organization](https://coreos.com/blog/introducing-operator-framework-metering).
+      The Metering Operator is a generic reporting tool to provide accountability for how resources are used across a cluster cherry picking and storing information from different sources. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster for storage. The Metering Operator is a part of the [Kubernetes Reporting organization](https://github.com/kube-reporting).
```
Comment (timflannagan1) [inline charts/metering-ansible-operator/values.yaml:230]:
```suggestion
      The Metering Operator is a generic reporting tool to provide accountability for how resources are used across a cluster cherry-picking and storing information from different sources. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster for storage. The Metering Operator is a part of the [Kubernetes Reporting organization](https://github.com/kube-reporting).
```

File: charts/metering-ansible-operator/values.yaml [hunk 3]
```diff
@@ -227,21 +227,25 @@ olm:
...
       ### Core capabilities
 
-      * **Chargeback/Showback** - Break down the reserved and utlized resources requested by applications.
+      * **Long term storage** - Break down the reserved and utlized resources requested by applications and store the information as long as needed.
 
       * **Pod, Namespace & Cluster Reports** - Built in reports exist to break down CPU and RAM in any way you desire.
-
-      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly
+ 
+      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly. Use this reports, for intance, as the basis for chargeback and showback.
```
Comment (timflannagan1) [inline charts/metering-ansible-operator/values.yaml:240]:
```suggestion
      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly. Use these reports, for instance, as the basis for chargeback and showback.
```

File: charts/metering-ansible-operator/values.yaml [hunk 4]
```diff
@@ -227,21 +227,25 @@ olm:
...
 
       * **Post-Processing** - Reports are generated in CSV format and stored in persistent storage for further post-processing. Use this to send reminder emails, integrate into your ERP system, or graph on a dashboard.
+      
+      * **Roll-up reports** - Reports can be used as a source for other reports, allowing for more complex queries that span longer periods than those supported by the data source.
 
       * **HTTP API** - Reports can be queried from an in-cluster HTTP API in addition to reading from persistent storage.
+      
+      * **Integration with AWS EC2 billing** - Connect operator metering to AWS to calculate EC2 costs of your resources.
```
Comment (timflannagan1) [inline charts/metering-ansible-operator/values.yaml:248]:
```suggestion
      * **Integration with AWS EC2 billing** - Connect metering to AWS to calculate EC2 costs of your resources.
```

File: charts/metering-ansible-operator/values.yaml (hunk 1 above)
Comment (bentito) [reply charts/metering-ansible-operator/values.yaml:239]: I think there is some github glitch here.  If you view the file in github the changes are rendered across the several commits that touch L240, properly

**Comment by timflannagan1** [comment]
Closing in favor of #1339 which was merged.

**Pull request closed without merging**